src/debugger.c

```diff
@@ -268,12 +270,12 @@ static void DEBUGHandleKeyEvent(SDL_Keycode key,int isShift) {
 
 	switch(key) {
 
-		case DBGKEY_STEP:									// Single step (F11 by default)
+		case DBGKEY_STEP:								// Single step (F11 by default)
 			currentMode = DMODE_STEP; 						// Runs once, then switches back.
 			break;
 
 		case DBGKEY_STEPOVER:								// Step over (F10 by default)
-			opcode = real_read6502(pc, false, 0);							// What opcode is it ?
+			opcode = real_read6502(pc, true, currentPCBank);			// What opcode is it ?
```

Comment:
This seemed like an obvious mistake too.  Fetching the opcode should read memory in debug mode, and we need to check the correct bank.